src/result_generation/height.py

```diff
@@ -119,12 +127,53 @@ def process_depthmaps(self):
 
         return depthmaps
 
+    def process_depthmaps_MCNN(self):
+        depthmaps_file = []
+        for artifact in self.artifacts:
+            input_path = self.get_input_path(
+                self.scan_directory, artifact['file'])
+            depthmaps_file.append(input_path)
+        scans = []
+        scans.append(depthmaps_file)
+        samples = list(
+            map(partial(preprocessing.sample_systematic_from_artifacts, n_artifacts=5), scans))
+        return samples
+
+    def create_multiartifact_sample(self, depthmap):
+        depthmaps = np.zeros((240, 180, 5))
+
+        for i, depthmap_path in enumerate(depthmap[0]):
+            data, width, height, depthScale, max_confidence = preprocessing.load_depth(
+                depthmap_path)
+            depthmap, height, width = preprocessing.prepare_depthmap(
+                data, width, height, depthScale)
+            depthmap = preprocessing.preprocess(depthmap)
+            depthmaps[:, :, i] = tf.squeeze(depthmap, axis=2)
+
+        depthmaps = tf.stack([depthmaps])
+        return depthmaps
+
     def run_height_flow(self):
         depthmaps = self.process_depthmaps()
         height_predictions = inference.get_height_predictions_local(depthmaps)
         generated_timestamp = datetime.now().strftime('%Y-%m-%dT%H:%M:%SZ')
         self.post_height_results(height_predictions, generated_timestamp)
 
+    def run_height_flow_MCNN(self):
+        depthmap = self.process_depthmaps_MCNN()
+        depthmap = self.create_multiartifact_sample(depthmap)
+        height_predictions = inference.get_MCNN_height_predictions_local(
+            depthmap)
+        generated_timestamp = datetime.now().strftime('%Y-%m-%dT%H:%M:%SZ')
```

Comment:
`%Y-%m-%dT%H:%M:%SZ` is now duplicated. We could define it as a global variable to avoid duplication